languages/go/exercises/concept/constants/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+In this exercise you will establish some constants to be used in the operation of a bank.
+
+You have five tasks:
+
+## 1. Represent the fixed interest rate on a loan
+
+Create the `FixedInterestRate` untyped numeric constant to hold the value of 5% (5/100), then implement the `GetFixedInterestRate` function to return it:
+
+```go
+GetFixedInterestRate()
+// Output: 0.05
+```
+
+## 2. Represent the number of days in a year
+
+Create the `DaysPerYear` constant with type `int` to hold the value 360, then implement the `GetDaysPerYear` function to return it:
```

Comment:
Aren't there 365 days in a year?

Reply:
Hi! I'm working on this now, sorry for the delay. 360 days is commonly used for interest calculation, at least in the USA.